kinto/core/storage/memory.py

```diff
@@ -46,9 +46,9 @@ def strip_deleted_record(self, collection_id, parent_id, record,
     def set_record_timestamp(self, collection_id, parent_id, record,
                              modified_field=DEFAULT_MODIFIED_FIELD,
                              last_modified=None):
-        timestamp = self._bump_timestamp(collection_id, parent_id, record,
-                                         modified_field,
-                                         last_modified=last_modified)
+        timestamp = self._bump_and_store_timestamp(collection_id, parent_id, record,
+                                                   modified_field,
+                                                   last_modified=last_modified)
```

Comment:
If the `MemoryBasedStorage` relies on  a `self._bump_and_store_timestamp()`, then every child class will have to implement it. So it should not be prefixed with `_`. And should raise `NotImplementedError` etc. :)

Reply:
Yes sure thing.